src/plugins/data/server/search/collectors/usage.ts

```diff
@@ -65,12 +65,12 @@ export function usageProvider(core: CoreSetup): SearchUsage {
   const trackSuccess = (duration: number) => {
     collectedUsage.successCount++;
     collectedUsage.totalDuration += duration;
-    return updateSearchUsage();
+    return updateSearchUsage()!;
```

Comment:
Why is this necessary?

Reply:
seems the `DebouncedFunc` is returing `Promise<void> | undefined`